db/models.js

```diff
@@ -133,4 +133,42 @@ exports.addCommentByArticleId = (articleId, commentContent) => {
         return result.rows;
       });
   };
-  
\ No newline at end of file
+  
+  exports.updateArticleById = (articleId, incVotesBy) =>{
+
+    const regex = /^[0-9]+$/g;
+    if (!regex.test(articleId)) {
+      const error = new Error("Invalid input type");
+      error.msg = "Invalid input type";
+      error.status = 400;
+      throw error;
+    }
+    const regex2 = /^[\d +-]+$/g;
```

Comment:
The variable naming here could be more semantic, rather than regex,regex2 and error, error2